recipes/paintera/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "paintera" %}
+{% set version = "0.17.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 71024837bbfe3a531559a7dbccaffc1f9fd08829ff96ca4d97d841abcf479ea9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv"
+  skip: True  # [py2k]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - jgo >=0.4.0
+    - paintera-conversion-helper
+
+test:
+  imports:
+    paintera
+
+about:
+  home: https://github.com/saalfeldlab/paintera-cmd
+  license: GPL2
+  license_family: GPL2
```

I think here it should be just
```suggestion
  license_family: GPL
```